### ISSUE TITLE
[Bug] Weapon gimbal doesn't work on older weapons

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#2850]
That applies to the weapons that have been made pre-20w35a.
Newly-placed weapons have working gimbal though.

Here's a video showing the issue:
[0XT2b2qWyD.zip](https://github.com/tsunamayo/Starship-EVO/files/5147157/0XT2b2qWyD.zip)

**Comment by tsunamayo** [comment]
@Kaiser-Indrasil could you send me the ship? What is the logic activating those weapon? There is a bug were gimbal wont work with rapid-fire kind of logic. It have to be plugged directly to the hotkey. Cheers

**Comment by Kaiser-Indrasil** [comment]
Yeah, this is exactly the way these weapons are linked to the yoke.
Here's the blueprint:
[Blueprints.zip](https://github.com/tsunamayo/Starship-EVO/files/5149935/Blueprints.zip)
